src/coreclr/src/zap/zapimage.cpp

```diff
@@ -2210,10 +2210,7 @@ ZapImage::CompileStatus ZapImage::TryCompileMethodWorker(CORINFO_METHOD_HANDLE h
 BOOL ZapImage::ShouldCompileMethodDef(mdMethodDef md)
 {
     DWORD partialNGenStressVal = PartialNGenStressPercentage();
-    if (partialNGenStressVal &&
-        // Module::AddCerListToRootTable has problems if mscorlib.dll is
-        // a partial ngen image
-        m_hModule != m_zapper->m_pEECompileInfo->GetLoaderModuleForMscorlib())
+    if (partialNGenStressVal)
```

Comment:
I don't have context on code change here

Reply:
This was workaround for Constrained Execution Regions that do not exist in CoreCLR.